espresso/src/main/resources/deviceInfo.json

```diff
@@ -0,0 +1,374 @@
+{
+  "redmi 10x 4g": {
+    "statusBarHeight": 100
+  },
+  "oppo a78 5g": {
+    "statusBarHeight": 64,
+    "navBarHeight": 88,
+    "deviceHeight": 1612
+  },
+  "samsung galaxy tab s8 5g": {
+    "statusBarHeight": 51,
+    "navBarHeight": 102,
+    "deviceHeight": 1600
+  },
+  "vivo vivo y22": {
+    "statusBarHeight": 60,
+    "navBarHeight": 79,
+    "deviceHeight": 1612
+  },
+  "samsung galaxy s22": {
+    "statusBarHeight": 81,
+    "navBarHeight": 144,
+    "deviceHeight": 2340
+  },
+  "google pixel 3": {
+    "statusBarHeight": 66
+  },
+  "google pixel 3a": {
+    "statusBarHeight": 66
+  },
+  "samsung galaxy s21 5g": {
+    "statusBarHeight": 80,
+    "navBarHeight": 144,
+    "deviceHeight": 2400
+  },
+  "google pixel 6 pro": {
+    "statusBarHeight": 145,
+    "navBarHeight": 56,
+    "deviceHeight": 3120
+  },
+  "samsung galaxy s10": {
+    "statusBarHeight": 112,
+    "navBarHeight": 126,
+    "deviceHeight": 2280
+  },
+  "oneplus6t": {
+    "statusBarHeight": 80
+  },
+  "google pixel 6": {
+    "statusBarHeight": 124,
+    "navBarHeight": 42,
+    "deviceHeight": 2400
+  },
+  "samsung galaxy note20 ultra 5g": {
+    "statusBarHeight": 92,
+    "navBarHeight": 135,
+    "deviceHeight": 2400
+  },
+  "google pixel 3 xl": {
+    "statusBarHeight": 171
+  },
+  "oneplus 7": {
+    "statusBarHeight": 80
+  },
+  "oppo reno3 pro": {
+    "statusBarHeight": 110
+  },
+  "samsung galaxy s23 ultra": {
+    "statusBarHeight": 75,
+    "navBarHeight": 135,
+    "deviceHeight": 2316
+  },
+  "realme  8": {
```

Comment:
there seem to be 2 spaces here. also check anywhere else if its the case

Reply:
it seems to be present in devices.csv. check line `39036`.